### PR TITLE
sprockets 4 support

### PR DESCRIPTION
**Pull request by ahorek**

### CODE REVIEW

File: spec/dummy/app/assets/config/manifest.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+//= link application.js
+//= link day.css
+//= link night.css
+//= link email.css
+//= link favicon.ico
```
Comment (glebm) [inline spec/dummy/app/assets/config/manifest.js]: Let's just link all the images:

```
//= link_tree ../images/
```

File: app/assets/config/thredded_manifest.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+//= link thredded.js
+//= link thredded.css
+//= link_directory ../images/thredded .svg
```
Comment (glebm) [inline app/assets/config/thredded_manifest.js]: There are some png images as well for the onebox:
https://github.com/thredded/thredded/tree/master/app/assets/images/favicons

File: spec/dummy/app/assets/config/manifest.js (hunk 1 above)
Comment (ahorek) [reply spec/dummy/app/assets/config/manifest.js]: done

File: app/assets/config/thredded_manifest.js (hunk 2 above)
Comment (ahorek) [reply app/assets/config/thredded_manifest.js]: thanks, done

**Pull request merged**